base/applications/cmdutils/more/lang/bg-BG.rc

```diff
@@ -3,13 +3,35 @@ LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
 STRINGTABLE
 BEGIN
     IDS_USAGE "Показва данните на екрана, разделени на страници.\n\n\
-  MORE < [Устройство:][Път]Файлово име\n\
-  Команда | MORE\n\
-  MORE [Устройство:][Път]Файлово име\n\n\
-  [Устройство:][Път]Файлово име    Файл, чието съдържание да бъде показано.\n\
-  Команда\t\t     Команда, чийто изход да бъде показан.\n\n\
-  При въпроса ""-- Продължаване --"" натиснете произволен клавиш, за показаване на следващата страница.\n"
+  MORE [options] < [Drive:][Path]Filename\n\
+  Command | MORE [options]\n\
+  MORE [options] [Drive:][Path]Filename\n\n\
+Options:\n\
+  /E        Enable extended features.\n\
+  /C        Clear screen before displaying page.\n\
+  /P        Expand form-feed characters.\n\
+  /S        Shrink multiple blank lines to a single line.\n\
+  /Tn       Expand tabs to n spaces (default: 8).\n\
+  +n        Start displaying the first file at line n.\n\n\
+  [Drive:][Path]Filename     A file, whose content shall be displayed.\n\
+  Command                    A command, whose output shall be displayed.\n\n\
+If extended features are enabled, the following commands are available\n\
+at ""-- Continue --"" prompt:\n\
+   P n      Display n lines.\n\
+   S n      Skip n lines.\n\
+   F        Display next file.\n\
+   Q        Quit.\n\
+   ?        Show help line.\n\
+   =        Show line number.\n\
+   <Space>  Display next page.\n\
+   <Enter>  Display next line.\n\
+   <Esc>    Same as Q.\n"
+
     IDS_CONTINUE "-- Продължаване --"
     IDS_CONTINUE_PROGRESS "-- Продължаване (%d%%) --"
     IDS_FILE_ACCESS "Няма достъп до файл %s.\n"
+    IDS_BAD_FLAG "Invalid argument - '%ls'\n"
+    IDS_CONTINUE_OPTIONS "-- Continue (%d%%) [Options: psfq=<Space><Enter><Esc>] --"
```

Comment:
Here the format of the prompts is compatible with MS ones.

Reply:
OK.